submission/persistence-sql/src/main/kotlin/ac/uk/ebi/biostd/persistence/mapping/extended/from/ToDbSubmissionMapper.kt

```diff
@@ -22,25 +22,30 @@ internal class ToDbSubmissionMapper(
     private val tagsRefRepository: TagDataRepository,
     private var userRepository: UserDataRepository
 ) {
-    fun toSubmissionDb(submission: ExtSubmission) = DbSubmission().apply {
-        accNo = submission.accNo
-        title = submission.title
-        status = getStatus(submission.status)
-        method = getMethod(submission.method)
-        version = submission.version
-        relPath = submission.relPath
-        rootPath = submission.rootPath
-        secretKey = submission.secretKey
-        creationTime = submission.creationTime
-        modificationTime = submission.modificationTime
-        releaseTime = submission.releaseTime
-        owner = getUser(submission.owner)
-        submitter = getUser(submission.submitter)
-        accessTags = toAccessTag(submission.collections)
-        tags = toTags(submission.tags)
-        released = submission.released
-        attributes = submission.attributes.mapIndexedTo(sortedSetOf(), ::toDbSubmissionAttribute)
-        rootSection = submission.section.toDbSection(this, ROOT_SECTION_ORDER)
+
+    fun toSubmissionDb(submission: ExtSubmission) = toSubmissionDb(submission, DbSubmission())
+
+    fun toSubmissionDb(submission: ExtSubmission, dbSubmission: DbSubmission): DbSubmission {
```

Comment:
I think it would be clearer is this fucntion is an extension like  `DbSubmission.toUpdatedSubmission(submission: ExtSubmission)` because, from the code, it seems weird to pass the current to a mapping method